eng/Build.props

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <ProjectToBuild Include="$(MSBuildThisFileDirectory)..\dir.proj" />
+    <ProjectToBuild Include="$(MSBuildThisFileDirectory)..\dir.proj" Condition="'$(SignPackages)' != 'true' and '$(PublishPackages)' != 'true'" />
+    <ProjectToBuild Include="$(MSBuildThisFileDirectory)..\signPackages.builds" Condition="'$(SignPackages)' == 'true'" />
```

Comment:
While this project selection could work I think it is better handled in build.proj using a custom target/property.